styles/stylesheet.css

```diff
@@ -0,0 +1,403 @@
+/*colors
+navy #083c60
+green #81b741
+light gray #d3d3d3
+dark gray #3d3d3d
+*/
+
+* {
+  box-sizing: border-box;
+}
+
+#core {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+header {
+  background-color: #083c60;
+  margin: 0 auto;
+}
+
+.fas, .fa-shopping-cart {
+  float: right;
+  color: #ffffff;
+}
+
+.shopping {
+  display: flex;
+  flex-flow: column;
+  float: right;
+  margin: .5em;
+}
+.checkout {
+  background-color: #81b741;
+  text-align: center;
+  border: 2px solid white;
+}
+
+.checkout-2 {
+  color: #ffffff;
+  text-align: center;
+}
+
+.logo-image {
+  width: 45%;
+  margin: .5em 0 0 .5em;
+}
+
+h1 {
+  font-family: 'Domine', serif;
+  font-size: 2.5em;
+  font-style: italic;
+  color: #81b741;
+  border: none;
+  text-align: left;
+  width: auto;
+  margin: .5em;
+}
+
+h2 {
+  font-family: 'Domine', serif;
+  font-size: 2em;
+  color: #083c60;
+  padding: .5em .5em .25em .5em;
+  border: none;
+  margin: .25em .5em .5em .5em;
+}
+
+h3 {
+  font-family: 'Domine', serif;
+  font-size: 1.5em;
+  color: #083c60;
+  padding: .25em;
+  border: none;
+  margin: .25em;
+}
+
+h4 {
+  font-family: 'Domine', serif;
+  font-size: 1.25em;
+  color: #083c60;
+  padding: .5em;
+  border: none;
+  margin: .5em;
+}
+
+a, div {
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
```

Comment:
Try not to use px for font or sizing anymore, they are not responsive, use em or % etc instead. Line border width is about the only place I would Use it.